Python/Product/Cookiecutter/Model/GitClient.cs

```diff
@@ -147,6 +147,12 @@ class GitClient : IGitClient {
             }
         }
 
+        private Dictionary<string, string> GetEnvironment() {
+            return new Dictionary<string, string>() {
+                { "PATH", Path.GetDirectoryName(_gitExeFilePath) },
```

Comment:
We should add this path to the current value of PATH, probably at the start, or else we may break other expectations about what it can find.